test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -280,115 +281,29 @@ $ kafka-verify format=avro sink=materialize.public.compaction_test_sink sort-mes
 {"before": null, "after": {"row": {"a": 1, "b": 1}}, "transaction": {"id": "1"}}
 {"before": null, "after": {"row": {"a": 2, "b": 2}}, "transaction": {"id": "1"}}
 
-# verify output of consistency topic with real-time timestamp bindings
```

Comment:
We _could_ try and verify the json output in the progress topic. But that's maybe for another day...

Reply:
I do intend to follow up with this -- if not in this PR, early next week.